### DISCUSSION TITLE
How to use scrollable_navigation_sheet with auto_route?

### DISCUSSION THREAD

**Discussion by AbdulBasitab** [fujidaiti/smooth_sheets#187] [Q&A] [answered]
Hi Everyone,

Has anyone implmented navigation sheet with auto_route so kindly share how to get it done because I am stuck on this.

Thank you

**Comment by fujidaiti** [comment]
[This comment](https://github.com/fujidaiti/smooth_sheets/issues/17#issuecomment-1937001384) may be helpful.

**Reply by AbdulBasitab** [reply]
Thanks @fujidaiti for the reply sure I will check out the comment and see if it is helpful.

**Comment by motucraft** [comment]
Is there a simple example of how to integrate `NavigationSheet` with auto_route? Or is auto_route not supported?

**Comment by fujidaiti** [comment, marked as answer]
@AbdulBasitab @motucraft 

Sorry for the delayed response! I added an example [here](https://github.com/fujidaiti/smooth_sheets/blob/main/example/lib/tutorial/paged_sheet_with_auto_route.dart).

**Reply by AbdulBasitab** [reply]
Thanks @fujidaiti

**Reply by motucraft** [reply]
@fujidaiti 
Thank you for your reply and the example provided. I was wondering if auto_route can also be used with NavigationSheet?

**Reply by fujidaiti** [reply]
@motucraft 

Yes! The example uses PagedSheet(called NavigationSheet in the previous version) and auto_route as a navigation framework.
